paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc

```diff
@@ -587,18 +583,10 @@ class ConvMKLDNNHandlerT
       post_operations.append_sum(sum_scale);
     }
 
-    if (fuse_activation == "hard_sigmoid") {
-      post_operations.append_eltwise(activation_scale,
-                                     dnnl::algorithm::eltwise_linear,
-                                     fuse_alpha,
-                                     fuse_beta);
-      post_operations.append_eltwise(
-          activation_scale, dnnl::algorithm::eltwise_clip, 0.0f, 1.0f);
-    } else if (fuse_activation != "") {
-      const auto activation_algorithm =
-          platform::AcquireActivationAlgorithm(fuse_activation);
-      post_operations.append_eltwise(
-          activation_scale, activation_algorithm, fuse_alpha, fuse_beta);
+    bool has_activation = !ctx.Attr<std::string>("fuse_activation").empty();
+    if (has_activation) {
+      paddle::platform::AppendActivation(
+          ctx, post_operations, activation_scale);
     }
```

Comment:
Could you move this if to `AppendActivation` as well? part 1/2
```suggestion
    paddle::platform::AppendActivation(
        ctx, post_operations, activation_scale);
```